Hrishi5111998.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/115489262/212279325-8055a0d9-a698-43f5-bf82-c07dd150de06.png",
```

Comment:
I recommend more of an image like this. What do you think?

![image](https://user-images.githubusercontent.com/51391473/212356612-82586bed-b827-45f4-a097-41942be39370.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/212356612-82586bed-b827-45f4-a097-41942be39370.png",
```

Reply:
> I recommend more of an image like this. What do you think?
> 
> ![image](https://user-images.githubusercontent.com/51391473/212356612-82586bed-b827-45f4-a097-41942be39370.png)

profile one? ya its look good